src/project/manifest/environment.rs

```diff
@@ -48,7 +48,7 @@ impl EnvironmentName {
     /// to read from an environment variable, otherwise it will fall back to default
     pub fn from_arg_or_env_var(arg_name: Option<String>) -> Self {
         if let Some(arg_name) = arg_name {
-            return EnvironmentName::Named(arg_name);
+            return EnvironmentName::from_str(&arg_name).unwrap();
```

Comment:
I dont thinks its appropriate to unwrap here. If the user specifies a name with an underscore this will crash.

Reply:
Fair enough :-)